src/js/view/controls/components/settings/menu.js

```diff
@@ -17,31 +17,31 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
     let visible;
     let active = null;
     const submenus = {};
+
     const settingsMenuElement = createElement(SettingsMenuTemplate());
-    const topbarElement = settingsMenuElement.querySelector('.jw-settings-topbar');
 
-    const closeButton = button('jw-settings-close', () => {
-        instance.close();
-    }, 'Close Settings', [cloneIcon('close')]);
-    closeButton.show();
-    closeButton.element().addEventListener('keydown', function(evt) {
-        if (evt.keyCode !== 9 || evt.shiftKey) {
-            return;
+    const closeOnEnter = function(evt) {
+        if (evt && evt.keyCode === 27) {
+            instance.close(true);
+            evt.stopPropagation();
         }
+    };
+    settingsMenuElement.addEventListener('keydown', closeOnEnter);
 
+    const closeButton = button('jw-settings-close', () => {
         instance.close();
-    });
-
-    topbarElement.appendChild(closeButton.element());
+    }, 'Close Settings', [cloneIcon('close')]);
 
-    const keyHandler = function(evt) {
-        if (evt && evt.keyCode === 27) {
-            instance.close();
-            evt.stopPropagation();
+    const closeOnButton = function(evt) {
+        if (evt.keyCode === 13 || (evt.keyCode === 9 && !evt.shiftKey)) {
```

Comment:
@hghazzi this is to close the menu if tabbing past the X icon ? @aliciahurst what are you thoughts ? I like it but i feel like it might be confusing for some users

Reply:
that was part of the AC